src/components/TextField/TextField.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import classNames from 'classnames';
+import _ from 'lodash';
+import { bindClassNames } from '../../util/style-helpers';
+import Validation from '../Validation/Validation';
+
+const boundClassNames = bindClassNames('TextField');
+
+const {
+	bool,
+	string,
+	func,
+	number,
+	object,
+	any,
```

Comment:
i don't mind trailing commas, but i know some really hate it, and i just don't like inconsistency